applicationconfig/client/src/samples/java/ConfigurationSets.java

```diff
@@ -0,0 +1,126 @@
+import com.azure.applicationconfig.ConfigurationClient;
+import com.azure.applicationconfig.ConfigurationClientCredentials;
+import com.azure.applicationconfig.models.ConfigurationSetting;
+import com.azure.applicationconfig.models.SettingSelector;
+import com.azure.common.annotations.Beta;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.List;
+
+/**
+ * Sample demonstrates how to use Azure Application Configuration to switch between "beta" and "production"
```

Comment:
Is there going to be a Readme.md or a doc where the scenario itself is going to be explained and snippets of code showed? 
or another sort of doc that will have the link to these samples?

Reply:
I have an issue #3292 and #3293 that should link to these.